common/copr_common/enums.py

```diff
@@ -137,3 +138,17 @@ class FailTypeEnum(with_metaclass(EnumType, object)):
             "git_checkout_error": 33,
             "srpm_build_error": 34,
            }
+
+
+class ActionObjectTypeEnum(str, Enum):
+    """
+    Enum for object_type attribute in Action model
+    """
+    COPR = "copr"
+    REPOSITORY = "repository"
+    BUILD = "build"
+    BUILDS = "builds"
+    COPR_CHROOT = "copr_chroot"
+    CHROOT = "chroot"
+    MODULE = "module"
+    NONE = "None"
```

Comment:
Is this abstraction actually useful?  It's just string mapping to string?

Reply:
The `Action.object_type` field was there just for informational purposes and didn't serve anything else. But now I used it for filtering the actions which belong to project and in the future with #2805 done, we will use this to identify whether the action belongs to Copr, Build, CHroot, etc., so I did this abstraction to prevent typos

Reply:
Hmm, I'm not convinced this is anyhow useful (we are not mapping string to int or vice-versa).